libchisel/src/remapimports.rs

```diff
@@ -27,143 +27,56 @@ impl ModulePreset for Translations {
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_externalCodeCopy"),
-                        ImportPair::new("ethereum", "externalCodeCopy"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_codeCopy"),
-                        ImportPair::new("ethereum", "codeCopy"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_getReturnDataSize"),
-                        ImportPair::new("ethereum", "getReturnDataSize"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_returnDataCopy"),
-                        ImportPair::new("ethereum", "returnDataCopy"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_create"),
-                        ImportPair::new("ethereum", "create"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_call"),
-                        ImportPair::new("ethereum", "call"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_callCode"),
-                        ImportPair::new("ethereum", "callCode"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_callDelegate"),
-                        ImportPair::new("ethereum", "callDelegate"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_callStatic"),
-                        ImportPair::new("ethereum", "callStatic"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_storageLoad"),
-                        ImportPair::new("ethereum", "storageLoad"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_log"),
-                        ImportPair::new("ethereum", "log"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_storageStore"),
-                        ImportPair::new("ethereum", "storageStore"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_revert"),
-                        ImportPair::new("ethereum", "revert"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_finish"),
-                        ImportPair::new("ethereum", "finish"),
-                    ),
-                    (
-                        ImportPair::new("env", "ethereum_selfDestruct"),
-                        ImportPair::new("ethereum", "selfDestruct"),
-                    ),
-                ]
-                .iter()
-                .cloned()
-                .collect();
+                let namespaces = vec!["env", "index"];
```

Comment:
I think Assemblyscript has a decorator to change the namespace. 

See [this](https://github.com/ewasm/assemblyscript-ewasm-api/pull/2/files#diff-03ea643b15c0986ca110e293df93a9d6R76):
```
@external("debug", "print64")
export declare function print64(value: i64): void;
```

Reply:
@axic you're right, for AssemblyScript you can use the decorator `@external("ethereum", "methodName")`

Reply:
Can we close this PR then?

It would be nice starting a guide on languages, perhaps in the `ewasm/design` repo for now.